### PR TITLE
[zh] sync job admission-controllers validating-admission-policy

### PR DESCRIPTION
**Pull request by asa3311**

### CODE REVIEW

File: content/zh-cn/docs/reference/access-authn-authz/validating-admission-policy.md [hunk 1]
```diff
@@ -210,8 +210,7 @@ otherwise the failures are ignored.
 否则这些失败将被忽略。
 
 <!-- 
-See [Audit Annotations: validation falures](/docs/reference/labels-annotations-taints/audit-annotations/#validation-policy-admission-k8s-io-validation_failure)
-for more details about the validation failure audit annotation.
+See [Audit Annotations: validation failures](/docs/reference/labels-annotations-taints/audit-annotations/#validation-policy-admission-k8s-io-validation-failure)
```
Comment (windsonsea) [inline content/zh-cn/docs/reference/access-authn-authz/validating-admission-policy.md:213]: 这是误删了吗

File: content/zh-cn/docs/concepts/workloads/controllers/job.md [hunk 2]
```diff
@@ -1611,6 +1611,15 @@ the Job status, allowing the Pod to be removed by other controllers or users.
 为了实现这一点，Job 控制器创建的 Pod 带有 Finalizer `batch.kubernetes.io/job-tracking`。
 控制器只有在 Pod 被记入 Job 状态后才会移除 Finalizer，允许 Pod 可以被其他控制器或用户移除。
 
+{{< note >}}
+<!--
+See [My pod stays terminating](/docs/tasks/debug-application/debug-pods) if you
+observe that pods from a Job are stucked with the tracking finalizer.
+-->
+如果你观察到来自作业的 Pod 因跟踪 Finalizer 而无法正常终止，
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/workloads/controllers/job.md:1619]:
```suggestion
如果你观察到来自 Job 的 Pod 因跟踪 Finalizer 而无法正常终止，
```

File: content/zh-cn/docs/concepts/workloads/controllers/job.md [hunk 3]
```diff
@@ -1611,6 +1611,15 @@ the Job status, allowing the Pod to be removed by other controllers or users.
...
+请参阅[我的 Pod 一直处于终止状态](/zh-cn/docs/tasks/debug-application/debug-pods)。
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/workloads/controllers/job.md:1620]:
```suggestion
请参阅[我的 Pod 一直处于终止状态](/zh-cn/docs/tasks/debug/debug-application/debug-pods/)。
```
en link is incorrect.

File: content/zh-cn/docs/reference/access-authn-authz/validating-admission-policy.md (hunk 1 above)
Comment (asa3311) [reply content/zh-cn/docs/reference/access-authn-authz/validating-admission-policy.md]: 不好意思，误删了，已恢复

File: content/zh-cn/docs/concepts/workloads/controllers/job.md [hunk 4]
```diff
@@ -1611,6 +1611,15 @@ the Job status, allowing the Pod to be removed by other controllers or users.
 为了实现这一点，Job 控制器创建的 Pod 带有 Finalizer `batch.kubernetes.io/job-tracking`。
 控制器只有在 Pod 被记入 Job 状态后才会移除 Finalizer，允许 Pod 可以被其他控制器或用户移除。
 
+{{< note >}}
+<!--
+See [My pod stays terminating](/docs/tasks/debug-application/debug-pods) if you
+observe that pods from a Job are stucked with the tracking finalizer.
+-->
+如果你观察到来自 Job 的 Pod 因跟踪 Finalizer 而无法正常终止，
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/workloads/controllers/job.md:1619]:
```suggestion
如果你发现来自 Job 的某些 Pod 因存在负责跟踪的 Finalizer 而无法正常终止，
```

**Pull request merged**